Upgrade branch 2.6.x using TemplateControl

gmethvin opened the pull request:
```
Updated with template-control on 2017-10-05T22:24:00.271Z
  **/build.properties:
    sbt.version=1.0.2
  **/plugins.sbt:
    addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.6.6")

```

marcospereira commented:
Depends on:

- https://github.com/flyway/flyway/issues/1741
- https://github.com/tototoshi/sbt-slick-codegen/pull/14 (merged but not released)

gmethvin committed:
Updated with template-control on 2017-12-21T20:59:17.173Z
  **/build.properties:
    sbt.version=1.0.4
  **/plugins.sbt:
    addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.6.10")

marcospereira commented:
Closing in favor of #52.
